st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -106,6 +109,28 @@ def match(self, action_alias_match_api, **kwargs):
             pecan.abort(http_client.BAD_REQUEST, str(e))
             return [self._match_tuple_to_dict(match) for match in e.matches]
 
+    @request_user_has_permission(permission_type=PermissionType.ACTION_ALIAS_HELP)
+    @jsexpose(arg_types=[str, str, int], body_cls=ActionAliasHelpAPI,
+    status_code=http_client.ACCEPTED)
+    def help(self, action_alias_help_api, **kwargs):
+        """
+            Get available help strings for action aliases.
+
+            Handles requests:
+                POST /actionalias/help
+        """
+        filter_ = action_alias_help_api.filter
+        pack = action_alias_help_api.pack
+        limit = action_alias_help_api.limit
+        offset = action_alias_help_api.offset
```

Comment:
The rest of the API endpoints expect `pack`, `limit` and `offset` to be sent as query parameters. We need to follow the same logic here too. Frankly, `filter` also looks like query parameter, but we would need to ensure we can pass regex string in query or came up with a way to escape it.